pages/linux/semanage.md

```diff
@@ -0,0 +1,24 @@
+# semanage
+
+> SELinux Policy Management tool.
+> More information: <https://linux.die.net/man/8/semanage>.
+
+- Output local customizations:
+
+`semanage -S {{store}} -o {{output_file}}`
+
+- Input local customizations:
+
+`semanage -S {{store}} -i {{input_file}}`
+
+- Manage booleans. Booleans allow the administrator to modify the confinement of processes based on his configuration:
```

Comment:
Perhaps this will read better?
```suggestion
- Manage booleans. Booleans allow the administrator to modify the confinement of processes based on the current configuration:
```